cube/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ else()
 endif()
 
 if(UNIX AND NOT APPLE) # i.e. Linux
-    include(FindPkgConfig)
+    find_package(PkgConfig)
```

Comment:
Why have find_package here and not in a more common place? (it looks like it is present here but not in the vulkaninfo CMakeLists.

Reply:
Because I am error.

Reply:
I thought cube/CMakeLists.txt was the top level CMakeLists.txt not sure why

Reply:
Fixed now.